wagtail/core/models/i18n.py

```diff
@@ -174,7 +174,7 @@ def copy_for_translation(self, locale):
 
         Note that the copy is initially unsaved.
         """
-        translated, child_object_map = _copy(self)
+        translated, child_object_map = _copy(self, exclude_fields=['index_entries'])
```

Comment:
This rings a bell, but am curious as to why the change was needed

Reply:
ah, are you hitting https://github.com/wagtail/wagtail-localize/issues/311 ?

Reply:
Yes! It seems the same problem.

Reply:
spoke with @kaedroho and we should filter out all generic relations (which would include `index_entries` and `postgres_index_entries`)
will try to add further notes on this, but it may not be until Wednesday due to some travel

Reply:
I already added `postgres_index_entries` to fix the postgres tests.
There is still a failing test, but it seems unrelated to this PR...

Reply:
the failing test is against Django's `main` branch which is an allowed failure. Unfortunately GitHub Actions doesn't have a nice way to mark expected failures..

Reply:
related issue for `excluded_fields` - #7735

Reply:
Good! Do I need to change anything else? Or just wait?

Reply:
Hey @andresmrm there should not be anything else to change.
Will re-review tomorrow. Thank you

Reply:
This may be addressed by #8049, which will automatically exclude ``GenericRelation`` during copy operations.